lib/blacklight/access_controls/enforcement.rb

```diff
@@ -23,6 +23,14 @@ def current_ability
         @current_ability || raise("current_ability has not been set on #{self}")
       end
 
+      def except(*)
+        super.tap { |new_builder| new_builder.current_ability = current_ability }
+      end
+
+      def append(*)
+        super.tap { |new_builder| new_builder.current_ability = current_ability }
+      end
+
```

Comment:
No, they don't do the same thing. One calls `append` (`super`) and the other calls `except`